src/VerifyEmailHelperInterface.php

```diff
@@ -28,7 +28,7 @@ interface VerifyEmailHelperInterface
     public function generateSignature(string $routeName, string $userId, string $userEmail, array $extraParameters = []): VerifyEmailSignatureComponents;
 
     /**
-     * Validate a signed URI and mark the user a verified.
```

Comment:
The bundle doesn't actually change the state of a user. That is handled in user land. Maker will provide an implementation for this. This may be something we could do within the bundle, but I haven't coded that part out yet internally.